### PR TITLE
Refactor advisers importer

### PR DESCRIPTION
**Pull request by andyhd**
Refactored the importer to this process:

1. Load the Excel spreadsheet into memory
2. Save each sheet to a temporary CSV file
3. Use COPY FROM to load the CSV file into the db
4. Translate the data to Django model tables using SQL
5. Geocode the postcodes and update the db

This is quite a lot faster than the previous method of parsing the Excel file
and generating Django models and adding them using the ORM.

### CODE REVIEW

File: laalaa/apps/advisers/importer.py [hunk 1]
```diff
@@ -82,215 +86,397 @@ class ImportProcess(Thread):
...
-        self.category_criminal_sheet = workbook.sheet_by_name(
-            'CAT OF LAW CRIME')
-        self.category_civil_sheet = workbook.sheet_by_name('CAT OF LAW CIVIL')
-        self.outreach_sheet = workbook.sheet_by_name('OUTREACH SERVICE')
+        self.sheets = {}
+        self.temp_dir = tempfile.mkdtemp()
 
-    def interrupt(self):
-        self._interrupt.set()
+        # if options.pop('single_transaction', False):
```
Comment (s-block) [inline laalaa/apps/advisers/importer.py]: do you still want this commented code?

Comment (andyhd) [inline laalaa/apps/advisers/importer.py]: Ah, I forgot to put that back in

**Comment by s-block** [comment]
:+1: great work. A+

**Comment by s-block** [comment]
:+1:

**Pull request merged**